.dockerignore

```diff
@@ -0,0 +1,3 @@
+client
+package-lock.json
```

Comment:
@tegsheeb You do not need the package-lock.json? I thought you mentioned you had issues when trying to run without it.

Reply:
ooh, It was the opposite. It should be ignored, because yarn will create its own lock file. Yarn will thrown an error if there is existing lock file for package.json.